custom_components/birthdays/__init__.py

```diff
@@ -15,13 +15,17 @@
 
 CONF_NAME = 'name'
 CONF_DATE_OF_BIRTH = 'date_of_birth'
+CONF_UNIQUE_ID = 'unique_id'
 CONF_ICON = 'icon'
+CONF_ATTRIBUTES = 'attributes'
 DOMAIN = 'birthdays'
 
 BIRTHDAY_CONFIG_SCHEMA = vol.Schema({
     vol.Required(CONF_NAME): cv.string,
     vol.Required(CONF_DATE_OF_BIRTH): cv.date,
+    vol.Optional(CONF_UNIQUE_ID, default=None): cv.string,
     vol.Optional(CONF_ICON, default='mdi:cake'): cv.string,
+    vol.Optional(CONF_ATTRIBUTES, default={}): vol.Schema({cv.string: cv.string}),
```

Comment:
Yes, that was the point of this if I remember right. In my HA instance I have added certain attributes to the birthdays to be able to fetch these values elsewhere.
I am using a markdown card that has icons, category and phone numbers in addition to the name and birthday countdown.

My config file looks like this:
```yaml
- unique_id: kristian
  name: "Kristian"
  date_of_birth: 1996-08-26
  icon: "mdi:account"
  attributes:
    phone: "12344321"
    category: "Self"

- unique_id: test
  name: "Test"
  date_of_birth: 1990-01-01
  icon: "mdi:human-male"
  attributes:
    phone: "12344321"
    category: "Family"
```

It might not be useful for everyone but it's pretty nice being able to set custom attributes for them.

Reply:
Really like it, nice!